packages/evolution-legacy/src/styles/survey/_question.scss

```diff
@@ -46,6 +46,29 @@ label strong, .apptr__form__label-standalone strong {
 
 }
 
+.apptr__form-join-next {
```

Comment:
You just have the `next` styes, will it work with previous too?

Reply:
Unfortunatly, it is not possible to select backward adjacent divs in css... Can only do it forward... Should the joinWith only work forward?

Reply:
But we can fix this by checking for previous AND next widget and convert all joinWith to next beforehand. Fixed!